spec/lib/jobs/upsert-node-job-spec.js

```diff
@@ -0,0 +1,50 @@
+// Copyright 2016, EMC, Inc.
+/* jshint node:true */
+
+'use strict';
+
+var uuid = require('node-uuid');
+
+describe('upsert-node-job', function(){
+    var job,
+    NodeUpsertor, 
+    taskId = uuid.v4(),
+    context = {},
+    options = {
+        target: "some:test:node",
+        data: { 
+            key1:{
+                ip: "123.456.789.012",
+                port: "12345",
+                key2: {
+                    key3: "value"
+                }
+            }
+        } 
+    },
+    sandbox = sinon.sandbox.create(),
```

Comment:
There is var in line 9, and all the variable are separated by comma. For grammar I don't need var here, but do you mean I should change it for another line of variable definition for style reason?